ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
+418.6 "Parthenos" sync / 1[56]:[^:]*:Athena:8303:/
+422.5 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+424.9 "Ray of Light" sync / 1[56]:[^:]*:Anthropos:82EE:/
+431.5 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+434.3 "Unnatural Enchainment" sync / 1[56]:[^:]*:Athena:82DF:/
+442.4 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+450.5 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+459.0 "--untargetable--" sync / 14:[^:]*:Athena:8556:/
+466.0 "Theos's Ultima (enrage)" sync / 1[56]:[^:]*:Athena:8556:/
+
+# P2 Pallas Athena -p 8682:1012.1 -ii 831A 831B 879F 832C 87A0 8334 8325 8323 833D 8792 833E 86FD 831F 8683 8320 8322 8340 8328
+1007.1 "--sync--" sync / 14:[^:]*:Pallas Athena:8682:/ window 1100,10
+1012.1 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:8682:/
+1024.5 "Palladian Grasp" sync / 1[56]:[^:]*:Pallas Athena:831[CD]:/
+1027.7 "Palladian Grasp" sync / 1[56]:[^:]*:Pallas Athena:831[CD]:/
+1040.8 "Gaiaochos" sync / 1[56]:[^:]*:Pallas Athena:8326:/
+1051.9 "Summon Darkness" sync / 1[56]:[^:]*:Pallas Athena:832F:/
+1063.8 "Ultima Ray" sync / 1[56]:[^:]*:Hemitheos:8330:/
+1065.1 "Missing Link" sync / 1[56]:[^:]*:Pallas Athena:8586:/
+1074.1 "Demi Parhelion" sync / 1[56]:[^:]*:Pallas Athena:8327:/
+1084.2 "Geocentrism" sync / 1[56]:[^:]*:Pallas Athena:832[AB9]:/
+1085.2 "Divine Excoriation" sync / 1[56]:[^:]*:Pallas Athena:832E:/
+1094.4 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:86F6:/
+1108.6 "The Classical Concepts" sync / 1[56]:[^:]*:Pallas Athena:8331:/
+1130.0 "Implode" #sync / 1[56]:[^:]*:Concept of (Earth|Water|Fire):8333:/
+1133.4 "Palladian Ray" sync / 1[56]:[^:]*:Pallas Athena:8324:/
+1143.8 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:8682:/
+1153.0 "Crush Helm" sync / 1[56]:[^:]*:Pallas Athena:8317:/
+1153.8 "Crush Helm 1" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1154.9 "Crush Helm 2" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1156.0 "Crush Helm 3" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1157.1 "Crush Helm 4" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1159.2 "Crush Helm Buster" sync / 1[56]:[^:]*:Pallas Athena:8319:/
+1173.2 "Caloric Theory" sync / 1[56]:[^:]*:Pallas Athena:8338:/
+1185.9 "Pyre Pulse" sync / 1[56]:[^:]*:Pallas Athena:833A:/
+1197.1 "Dynamic Atmosphere" sync / 1[56]:[^:]*:Pallas Athena:833B:/
+1197.3 "Pyre Pulse" sync / 1[56]:[^:]*:Pallas Athena:833A:/
+1207.6 "Ekpyrosis" sync / 1[56]:[^:]*:Pallas Athena:831E:/
+1215.4 "Ekpyrosis" sync / 1[56]:[^:]*:Pallas Athena:8(321|683):/
+1226.8 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:8682:/
+1241.0 "Pangenesis" sync / 1[56]:[^:]*:Pallas Athena:833F:/
+1249.1 "Pantheos" sync / 1[56]:[^:]*:Pallas Athena:8342:/
+1254.9 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/ window 1.5,1.5
+1260.0 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/ window 1.5,1.5
+1265.0 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/ window 1.5,1.5
```

Comment:
```suggestion
1254.9 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/
1260.0 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/
1265.0 "Astral/Umbral Advent" sync / 1[56]:[^:]*:Hemitheos:834[34]:/
```

These are 5s apart, so shouldn't need a window.  Usually I would consider a shorter window if it's in the ~2.5s (maybe 3.0s if they're inconsistent) range.  If less I'd comment out the syncs, and if more I'd just leave them default window.